src/main/kotlin/com/dotronglong/faker/service/plugin/DelayResponsePlugin.kt

```diff
@@ -16,7 +15,7 @@ class DelayResponsePlugin : Plugin {
         return Mono.create { s ->
             val duration = (arguments?.get("duration") as Int?) ?: 0
             if (duration > 0) {
-                GlobalScope.launch {
+                runBlocking {
```

Comment:
Hi @snicoll I am a little bit new with this, is there truly benefit to use this instead of GlobalScope?

Reply:
The usage of global scope is [generally not recommended](https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines/-global-scope/) and generates warnings. That said, maybe my proposal is not optimal. Feel free to discard this change or to replace it with something better. Usually when creating `Mono` from Coroutines we tend to use https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-reactor/kotlinx.coroutines.reactor/mono.html.

Reply:
Thank you for your explanation. Do you think we should use coroutineScope https://kotlinlang.org/api/kotlinx.coroutines/kotlinx-coroutines-core/kotlinx.coroutines/coroutine-scope.html instead? Would you mind to help if it is a better solution. Or we can ignore this change because I will need time to try coroutine. Really appreciate if you can propose a better solution. I believe 4 years ago GlobalScope is a good option for me at that time, however, it may not be at this moment.